sources/android2/settings.gradle

```diff
@@ -1 +1,2 @@
 include ':KalturaClient'
+include ':DemoApplication'
```

Comment:
DemoApplication should be added only to OVP project and therefore should be mentioned only under tests/ovp/android2/...

Reply:
removed